docs/types/flusher.md

```diff
@@ -0,0 +1,14 @@
+# Flushers
+
+Flushers handle reading entries from buffers in chunks, flushing them to their final destination,
```

Comment:
A little more context on why someone might want to configure a flusher would be helpful.